### PR TITLE
font awesome icons

### PR DESCRIPTION
**Pull request by FredrikLastow**
Added font awesome icons files and removed old icon files
Implemented new font awesome icons
Fixed messages navbar bug on ios and android

### CODE REVIEW

File: www/css/font-awesome.css [hunk 1]
```diff
@@ -0,0 +1,2337 @@
+/*!
```
Comment (navro) [inline www/css/font-awesome.css]: We don't need this file, `.min` should be enough

File: www/css/index.css [hunk 2]
```diff
@@ -6,15 +6,42 @@ a:active {
...
+  font-size: 24px;
+}
+
+.tabbar i.fa-comments{
+  font-size: 30px;
+}
+
+.tabbar i.fa-home{
+  padding-top: 1px;
+  font-size: 32px;
```
Comment (navro) [inline www/css/index.css]: Is this icon much smaller than the others? Or why does it have to be 8px larger than the others

Comment (FredrikLastow) [inline www/css/index.css]: Yep, its very small for some reason

Comment (navro) [inline www/css/index.css]: How stupid

File: www/messages.html [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
-        <a href="#" class="back link">
-          <i class="icon icon-back"></i>
-        </a>
-      </div>
-      <div class="center sliding" id="group-name"></div>
+      {{#unless @global.android}}
+        <div class="left sliding">
+          <a href="#" class="back link"> 
+            <i class="icon icon-back"></i>
+            <span>Back</span>
```
Comment (navro) [inline www/messages.html]: Tillbaka

File: www/messages.html [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
+          </a>
+        </div>
+      {{/unless}}
+      <div class="center sliding" id="messages-group-name">Spindelchatt</div>
```
Comment (navro) [inline www/messages.html]: Should be set in js

Comment (FredrikLastow) [inline www/messages.html]: it is, just a sample text ^^ Will remove it though

File: www/css/index.css [hunk 5]
```diff
@@ -6,15 +6,42 @@ a:active {
...
+
 .tabbar .badge {
   background: #ffe0b3;
-  color: #eb7125
+  color: #eb7125;
+  border: 1px solid #eb7125;
+  position: absolute;
+  top: 0;
+  margin-left: 17px;
+  margin-top: 2px;
```
Comment (navro) [inline www/css/index.css]: Can we add some more height to this? I think it should be more square

**Pull request merged**